model/Core/Vocabularies/RelationshipType.md

```diff
@@ -4,11 +4,11 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-TODO
+A reasonable estimation of the relation between two identified Elements from a developer perspective.
```

Comment:
Same comments as above - prefer to use the 2.3 spec wording

Reply:
done